### PR TITLE
My transcriptions

### PR DESCRIPTION
**Pull request by thehabes**
Handles #46 .  The /projects servlet (GetAllUserProjects.java) now adds which interface to use for the project to the data nodes and may be what we need for #23 .

This branch is deployed at http://paleo.rerum.io.  Login with `bhaberbe@slu.edu` `3`

### CODE REVIEW

**Comment by thehabes** [comment]
Still leverages index.jsp for the login stuff.  Positioned to completely replace it, but probably not until we get our workflows figured out.

File: Source Packages/java/textdisplay/Project.java [hunk 1]
```diff
@@ -1683,5 +1683,24 @@ public void setSchemaURL(Connection conn, String newURL) throws SQLException {
       }
    }
    
+   public String mintInterfaceLinkFromFolio(int folioNum) throws SQLException{
+        Folio f = new Folio(folioNum);
```
Comment (cubap) [inline Source Packages/java/textdisplay/Project.java:1687]: We may want a possibility for a default number ([0] canvas, for example), but I'm not going to stop this.

**Comment by thehabes** [comment]
Great.  Just deployed this branch with the 2 PRs in it at http://paleo.rerum.io.  Login `bhaberbe@slu.edu` `3` to test the my transcription stuff.

**Review by cubap — APPROVED** [review]
This got a little crowded, but I think we're all square. I'll go check the deploy as well.

**Pull request merged**